data/ResourceUtil.cpp

```diff
@@ -1,104 +1,106 @@
-//this file was auto-generated by res2h
+// this file was auto-generated by res2h
 
 #include "Resources.h"
 
-const size_t res2hNrOfFiles = 46;
+const uint32_t res2hNrOfFiles = 47;
```

Comment:
1. It's completly normal to have a hardcoded default image for games without boxart

The theme can also add a custom image for games without boxart with the following syntax as specified in the GRIDTHEME.md : 
```xml
// Replace the missing boxart texture
<image name="missing_boxart">
	<path>new_path</path>
</image>
```

2. You answered it by yourself.

The thing is the default image for games without boxarts is part of the gridview, should I also include it in the other pull request with the rs2h update ?